babel.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  presets: [
+    '@babel/preset-env',
+    '@babel/preset-react',
+  ],
+  plugins: ['@babel/plugin-proposal-class-properties'],
```

Comment:
I don't actually know what this line does. I only added there because I saw it somewhere.